### PR TITLE
[4.0] Namespace administrator components com_banners

### PR DESCRIPTION
**Pull request by brianteeman**
continuing the work to namespace joomla4 one admin component at a time.

please wait for drone to successfully complete before reviewing.

### CODE REVIEW

File: administrator/components/com_banners/Model/TracksModel.php [hunk 1]
```diff
@@ -14,6 +14,10 @@
 use Joomla\CMS\Component\ComponentHelper;
 use Joomla\CMS\MVC\Model\ListModel;
 use Joomla\Component\Banners\Administrator\Helper\BannersHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Fileystem\File;
```
Comment (Quy) [inline administrator/components/com_banners/Model/TracksModel.php]: Spelling `Fileystem`

Comment (brianteeman) [reply administrator/components/com_banners/Model/TracksModel.php]: good spot - fixed

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**